ceph.spec.in

```diff
@@ -809,15 +809,13 @@ mkdir -p %{_localstatedir}/run/ceph/
 %{_mandir}/man8/ceph-clsinfo.8*
 %{_mandir}/man8/librados-config.8*
 #set up placeholder directories
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/tmp
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/mon
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/osd
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/mds
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/bootstrap-osd
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/bootstrap-mds
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/bootstrap-rgw
-%attr(770,ceph,ceph) %dir %{_localstatedir}/run/ceph/
```

Comment:
The problem might still occur with the removal. If the package is not owned by anything, it won't be cleaned up when the packages are removed which is generally considered a bad practice in the packaging world.